src/accordion.test.jsx

```diff
@@ -0,0 +1,185 @@
+	});
+
+});
+
+describe('AccordionPanel', function() {
+	let panel,
+		openPanel,
+		panelCustomIcon,
+		panelLeftIcon;
+
+	const customIcon = 'plus';
+	const customIconActive = 'minus';
+
+	beforeEach(() => {
+
+		panel = TestUtils.renderIntoDocument(
+			<AccordionPanel
+				label='First Section'
+				panelContent={
+					<div className='runningText'>
+						<p>Contrary to popular belief, Lorem Ipsum is not simply random text. It has roots in a piece of classical Latin literature from 45 BC, making it over 2000 years old. Richard McClintock, a Latin professor at Hampden-Sydney College in Virginia, looked up one of the more obscure Latin words, consectetur, from a Lorem Ipsum passage, and going through the cites of the word in classical literature, discovered the undoubtable source.</p>
+					</div>
+				} />
+		);
+
+		openPanel = TestUtils.renderIntoDocument(
+			<AccordionPanel
+				isOpen
+				label='First Section'
+				panelContent={
+					<div className='runningText'>
+						<p>Contrary to popular belief, Lorem Ipsum is not simply random text. It has roots in a piece of classical Latin literature from 45 BC, making it over 2000 years old. Richard McClintock, a Latin professor at Hampden-Sydney College in Virginia, looked up one of the more obscure Latin words, consectetur, from a Lorem Ipsum passage, and going through the cites of the word in classical literature, discovered the undoubtable source.</p>
+					</div>
+				} />
+		);
+
+		panelLeftIcon = TestUtils.renderIntoDocument(
+			<AccordionPanel
+				label='First Section'
+				iconAlign='left'
+				panelContent={
+					<div className='runningText'>
+						<p>Contrary to popular belief, Lorem Ipsum is not simply random text. It has roots in a piece of classical Latin literature from 45 BC, making it over 2000 years old. Richard McClintock, a Latin professor at Hampden-Sydney College in Virginia, looked up one of the more obscure Latin words, consectetur, from a Lorem Ipsum passage, and going through the cites of the word in classical literature, discovered the undoubtable source.</p>
+					</div>
+				} />
+		);
+
+		panelCustomIcon = TestUtils.renderIntoDocument(
+			<AccordionPanel
+				label='First Section'
+				iconAlign='left'
+				iconShape={customIcon}
+				iconShapeActive={customIconActive}
+				panelContent={
+					<div className='runningText'>
+						<p>Contrary to popular belief, Lorem Ipsum is not simply random text. It has roots in a piece of classical Latin literature from 45 BC, making it over 2000 years old. Richard McClintock, a Latin professor at Hampden-Sydney College in Virginia, looked up one of the more obscure Latin words, consectetur, from a Lorem Ipsum passage, and going through the cites of the word in classical literature, discovered the undoubtable source.</p>
+					</div>
+				} />
+		);
+	});
+	afterEach(() => {
+		panel = null;
+		openPanel = null;
+	});
+
+	it(`has the class ${ACTIVEPANEL_CLASS} when state is open`, function() {
```

Comment:
Given that the panel content visibility is based on `height` manipulation rather than CSS class toggling, we'll have some trouble checking this in tests.

Our thinking was that covering state changes and the `ACTIVEPANEL_CLASS` would be adequate for alerting us when content visibility might break